pkg/scheduler/api/node_info.go

```diff
@@ -580,3 +586,24 @@ func (ni *NodeInfo) SubGPUResource(pod *v1.Pod) {
 		}
 	}
 }
+
+// getUnhealthyGPUs returns all the unhealthy GPU id.
+func (ni *NodeInfo) getUnhealthyGPUs(node *v1.Node) (unhealthyGPUs map[int]bool) {
+	unhealthyGPUs = map[int]bool{}
+	devicesStr, ok := node.Annotations[UnhealthyGPUIDs]
```

Comment:
BTW, @william-wang @shinytang6 Does device plugin report the unhealthy GPU ids? And these ids will be patched to node object as an annotation?

Reply:
The device plugin patch the unhealthy gpu id.  We need to make clear the following pr in device plugin firstly and then work on this one.
https://github.com/volcano-sh/devices/pull/24

Reply:
@william-wang volcano-sh/devices#24 have been merge。 Anyother questions in this pr.